Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -439,7 +439,30 @@ std::string HiresTexture::GetTextureDirectory(const std::string& game_id)
 
   // If there's no directory with the region-specific ID, look for a 3-character region-free one
   if (!File::Exists(texture_directory))
+  {
+    // First try looking if a text file exists with either the region-specific ID or 3-character
+    // region-free ID and check if it contains a referral to texture pack directory
+    if (File::Exists(File::GetUserPath(D_HIRESTEXTURES_IDX) + game_id + ".txt") ||
+        File::Exists(File::GetUserPath(D_HIRESTEXTURES_IDX) + game_id.substr(0, 3) + ".txt"))
+    {
+      std::string texture_textFile = File::GetUserPath(D_HIRESTEXTURES_IDX) + game_id + ".txt";
+      if (File::Exists(File::GetUserPath(D_HIRESTEXTURES_IDX) + game_id.substr(0, 3) + ".txt"))
+        texture_textFile = File::GetUserPath(D_HIRESTEXTURES_IDX) + game_id.substr(0, 3) + ".txt";
+
+      std::ifstream texture_textFile_stream(texture_textFile);
```

Comment:
You need to use `File::OpenFStream` instead of passing a string to the constructor, otherwise this won't work if the user has non-ASCII characters in the path and is on Windows.

Also, to follow our coding style, replace `textFile` with `text_file`.

Reply:
Ok, fair enough. I think I might have missed a files guidelines here and there, being my first time. I try to adjust the issues.

Reply:
I think I managed to update my pull request.